e2e-tests/portal/cypress/e2e/journeys/checker/issued-specs-new/submit-MIN-deal-with-issued-facility/submit-MIN-deal-with-issued-facilities.spec.js

```diff
@@ -68,19 +68,10 @@ context('A maker issues facilities, submits to checker; checker submits deal to
     fillAndSubmitIssueLoanFacilityForm();
 
     // check facility statuses are correct
-    bondRow
-      .bondStatus()
-      .invoke('text')
-      .then((text) => {
-        expect(text.trim()).to.equal('Completed');
-      });
 
-    loanRow
-      .loanStatus()
-      .invoke('text')
-      .then((text) => {
-        expect(text.trim()).to.equal('Completed');
-      });
+    cy.assertText(bondRow.bondStatus(), 'Completed');
```

Comment:
is there a const for these?

Reply:
I'm not sure, but status constants (and/or usage of them) should be handled separately. This PR is purely E2E, introducing an "assert text" cypress command

Reply:
tech debt ticket for all status strings to come from constants please